### PR TITLE
Release 5.1.0

### PR DESCRIPTION
**Pull request by pgrzesik**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,12 @@
 
 All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.
 
+## [5.1.0](https://github.com/serverless/utils/compare/v5.0.0...v5.1.0) (2021-05-19)
+
+### Features
+
+- Add `download` util ([#84](https://github.com/serverless/utils/pull/84)) ([888a583](https://github.com/serverless/utils/commit/888a58328099f1860fac8305fe32bb05325f039a)) ([Piotr Grzesik](https://github.com/pgrzesik))
```
Comment (medikoo) [inline CHANGELOG.md:9]: I think it's more a fix than a feature.

It'll be also to improve description, e.g.: _Replace `download` dependency with patched locally version that fixes internal timeout handling_

Comment (medikoo) [reply CHANGELOG.md:9]: Sorry @pgrzesik I take back. It's a feature, as we introduce a module here, and it's externally where we will replace `download` with this one.

(when writing I blindly assumed it's just internal switch)

**Review by medikoo — APPROVED** [review]
👍

File: CHANGELOG.md (hunk 1 above)
Comment (pgrzesik) [reply CHANGELOG.md:9]: :+1: It's going to be marked as `fix` in `serverless/serverless`

**Pull request merged**